Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
+
+class IdealWeight
```

Comment:
correct the name in accordance with these recommendations
https://medium.com/selleo/a-subjective-guide-to-naming-stuff-in-ruby-on-rails-classes-b44928b6c49a

Reply:
I corrected it